index.bs

```diff
@@ -7716,8 +7715,7 @@ platform objects for {{DOMException|DOMExceptions}}.
 
 <h4 id="es-DOMException">DOMException</h4>
 
-IDL {{DOMException}} values are represented
-by platform objects for {{DOMException|DOMExceptions}}.
+IDL {{DOMException}} values are represented by [=platform objects=].
```

Comment:
Comparing this sentence to the rest of the ones in this section, it makes very little sense. I would just remove it. Or maaaybe replace it with "IDL {{DOMException}} values are represented by ECMAScript Object values" (cf. interfaces).